src/Symfony/Bundle/TwigBundle/Resources/views/Exception/logs.html.twig

```diff
@@ -1,5 +1,19 @@
 {% set channel_is_defined = (logs|first).channel is defined %}
 
+<script>
+    function toggleLevel(name) {
+        var rows = document.querySelectorAll('table.logs tr[data-log-level="' + name + '"]');
+        for (var i = 0, row; row = rows[i]; ++i) {
+            row.style.display = row.style.display == 'none' ? 'table-row' : 'none';
+        }
+    }
+</script>
+<div>
+    {% for level in ['DEBUG', 'INFO', 'NOTICE', 'WARNING', 'ERROR', 'CRITICAL', 'ALERT', 'EMERGENCY'] %}
```

Comment:
this shall not be hardcoded levels here.
i do know it's not likely to happen, but if one come up with other log level like "CONFIDENT" or "PERFORMANCE", he will need to be aware that he has to update template as well. not easy thing

Reply:
"he" will be SF core :) not our problem. Tend to be pragmatic here :) (for a poc at least). The PSR levels are not likely to change.

Yes. Your point is valid; open for now.

Reply:
it's long term issue. PSR could be change after 2 years, and nobody will remember after 2 years that some template has to be change due to that